### PR TITLE
Upgrade to Vaadin 10.0.13

### PR DESCRIPTION
**Pull request by tulioag**

### CODE REVIEW

File: start-site/src/main/resources/application.yml [hunk 1]
```diff
@@ -1292,4 +1292,4 @@ spring:
     restart:
       additional_paths:
         - start-client/build/
-      additional_exclude: "**/*.js,**/*.css"
\ No newline at end of file
+      additional_exclude: "**/*.js,**/*.css"
```
Comment (snicoll) [inline start-site/src/main/resources/application.yml]: please don't change unrelated lines.

Comment (tulioag) [reply start-site/src/main/resources/application.yml]: Done

**Comment by snicoll** [comment]
@tulioag thank you for making your first contribution to start.spring.io. The update will be available in production shortly.
